src/components/general/BookmarksManager/BookmarkSideSheet/SideSheetManager/index.tsx

```diff
@@ -50,7 +52,7 @@ const createGroupByContextId = (allBookmarks: BookmarkListResponse[]) => {
 const createGroupByContext = (allBookmarks: BookmarkListResponse[]) => {
     const groupedContext: Record<string, BookmarkListResponse[]> = {};
     allBookmarks.reduce((__prev, curr) => {
-        const key = curr.context ? curr.context.id : 'unknown';
+        const key = curr.context ? curr.context.id : 'Global';
```

Comment:
when checking, use either constant or enum

Reply:
extracted the string into a const